Not in virtualenv
cuDNN not found!
bazel not found
{'label': 'programming_language', 'name': 'python', 'version': '3.8', 'verdict': 'yes', 'post_id': ['68917678']}
{'label': 'programming_language', 'name': 'python', 'version': '3.9', 'verdict': 'yes', 'post_id': ['68917678']}
......................................................................................
Current library: numpy (v1.21.6)
Knowledge base: {'library': {}, 'hardware': {}, 'programming_language': {'python': [('3.8', 'yes', ['68917678']), ('3.9', 'yes', ['68917678'])]}, 'operating_system': {}, 'tool': {}, 'database': {}, 'software': {}}
bottle is not installed in local
unknown
unknown
unknown
{'label': 'hardware', 'name': 'cuda', 'version': '9', 'verdict': 'yes', 'post_id': ['49465509', '47195590', '48571864', '48685342', '49164204', '49395649', '57681865']}
......................................................................................
Current library: tensorflow (v1.5)
Knowledge base: {'library': {}, 'hardware': {'cuda': [('9', 'yes', ['49465509', '47195590', '48571864', '48685342', '49164204', '49395649', '57681865'])]}, 'programming_language': {}, 'operating_system': {}, 'tool': {}, 'database': {}, 'software': {}}
--------------------------------DC ISSUE-----------------------------------------
ERROR: Local tensorflow (v1.5) is not compatible with cuda (v11.7) (0)
-----------------------------------END-------------------------------------------
